### PR TITLE
Corrected Grammer Mistake and Errors in RELEASE.md file

### PR DESCRIPTION
**Pull request by gargarchit**
Corrected Grammer Mistake and Errors in RELEASE.md file

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
 # Release
 
-To release new version of Kuma follow the steps:
+To release a new version of Kuma follows the steps:
```
Comment (yskopets) [inline RELEASE.md]: I think, the original `follow the steps` might be the right version.

Maybe we could change it to `follow the steps below` instead ?

File: RELEASE.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 
 1. Create a CHANGELOG.md.
-2. Double-check with changelog that all new features are documented on kuma.io website.
+2. Double-check with a changelog that all new features are documented on the kuma.io website.
```
Comment (yskopets) [inline RELEASE.md]: I think, `with the changelog` might be the right version.

We could also provide a link to the GitHub repo of kuma.io. E.g.,`documented on the [kuma.io website](https://github.com/Kong/kuma-website)`

File: RELEASE.md [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
 4. Create a new git tag.
-5. Push git tag. This will trigger the release job on CI.
+5. Push the git tag. This will trigger the release job on CI.
 6. Make sure that new binaries are available in [Bintray](https://bintray.com/kong/kuma).
-7. Download the new Kuma version and double check that it works with demo app. Check that is works both in `universal` and `kubernetes` modes.
+7. Download the new Kuma version and double-check that it works with the demo app. Check that is works both in `universal` and `kubernetes` modes.
 8. Merge PR to website repository.
 9. Create a new [Github release](https://github.com/Kong/kuma/releases)
-10. Announce new version on Kuma Slack #news channel.
+10. Announce a new version on the Kuma Slack #news channel.
```
Comment (yskopets) [inline RELEASE.md]: I think, `Announce the new version` might be the right version

File: RELEASE.md [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
 
 ## Major releases
 For major releases make sure that you also:
 
 1. Create a blog post on Kong's blog.
-2. Send newsletter about new release.
\ No newline at end of file
+2. Send a newsletter about a new release.
```
Comment (yskopets) [inline RELEASE.md]: I think, `a newsletter about the new release` might be the right version

File: RELEASE.md (hunk 1 above)
Comment (gargarchit) [reply RELEASE.md]: Thank You @yskopets  for review, Yeah I am changing this

File: RELEASE.md (hunk 2 above)
Comment (gargarchit) [reply RELEASE.md]: Corrected

File: RELEASE.md (hunk 3 above)
Comment (gargarchit) [reply RELEASE.md]: Changed

File: RELEASE.md (hunk 4 above)
Comment (gargarchit) [reply RELEASE.md]: changed

**Comment by gargarchit** [comment]
@yskopets Please Check changes made by me

**Comment by gargarchit** [comment]
Thanks, @yskopets I hope this pr is ready for merge

**Comment by yskopets** [comment]
@gargarchit Yeah, it's all good now.

Just waiting for CI build to finish.

**Comment by gargarchit** [comment]
Thanks @yskopets

**Pull request merged**